zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/Helium.java

```diff
@@ -411,7 +412,7 @@ public HeliumPackageSuggestion suggestApp(Paragraph paragraph) {
    * @return ordered list of enabled buildBundle package
    */
   public List<HeliumPackage> getBundlePackagesToBundle() {
-    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfoWithoutRefresh();
+    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfo();
```

Comment:
I updated the last comment. 

> And the performance hit is not that bad

If we change visualization order, it will call `clearNotExistsPackages` and it will fetch new helium.json, but what if the network is not good? 

Currently (master branch), `helium.json` will be fetched when users open the `#/helium` page. IMO it's enough. If users have serious problems due to synchronization or inconsistency in helium packages, we can improve it.